### PR TITLE
Rel 5 0 ut restore database part7

### PR DESCRIPTION
**Pull request by s7design**
Hi @mgruner 

One more PR with updated unit tests. I implemented some your sugestion from the preius commit about clean-up cache data and RestoreDatabase in PGP.t even though there is PublicKeyDelete() in test

Regards
Zoran

### CODE REVIEW

File: scripts/test/PID.t [hunk 1]
```diff
@@ -14,7 +14,6 @@ use vars (qw($Self));
 
 # get needed objects
 my $HelperObject = $Kernel::OM->Get('Kernel::System::UnitTest::Helper');
```
Comment (mgruner) [inline scripts/test/PID.t]: No RestoreDatabase here?

Comment (s7design) [inline scripts/test/PID.t]: Yes we could add here, the same reason even though there is PIDDelete. I will update it right now.

**Comment by s7design** [comment]
I updated PR

Regards
Zoran

**Comment by mgruner** [comment]
Merged, thanks!

**Pull request closed without merging**